### PR TITLE
leijurv go away

### PR DESCRIPTION
**Pull request by sourTaste000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,24 +27,20 @@
...
 
 For 1.15.2, [click here](https://www.youtube.com/watch?v=j1qKtCZFURM) and see description. If you need Forge 1.15.2, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.5.3), follow the instructions, and get the `api-forge` jar.
 
-This project is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
-the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 through 1.15.2. Baritone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
+This project is an updated version of [Baritone](https://github.com/cabaletta/baritone),
+the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 through 1.15.2. trombone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
```
Comment (dominikaaaa) [inline README.md:40]: if you're gonna update this you need to remove the 
> the original version of the bot for 1.8.9.....

File: README.md [hunk 2]
```diff
@@ -27,24 +27,20 @@
...
 
 Have committed at least once a day from Aug 1 2018 to Aug 1 2019.
 
-1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2
```
Comment (dominikaaaa) [inline README.md:46]: Keep this please

Comment (dominikaaaa) [reply README.md:46]: or add mine beside it and make it clear the top one is the original baritone btc

File: README.md [hunk 3]
```diff
@@ -80,14 +76,14 @@ BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAnd
 
 # FAQ
 
-## Can I use Baritone as a library in my custom utility client?
+## Can I use trombone as a library in my custom utility client?
```
Comment (dominikaaaa) [inline README.md:79]: capitalised

File: README.md [hunk 4]
```diff
@@ -80,14 +76,14 @@ BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAnd
...
 
 That's what it's for, sure! (As long as usage is in compliance with the LGPL 3.0 License)
 
 ## How is it so fast?
 
 Magic. (Hours of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
 
-## Why is it called Baritone?
+## Why is it called trombone?
```
Comment (dominikaaaa) [inline README.md:87]: capital, again

File: README.md [hunk 5]
```diff
@@ -80,14 +76,14 @@ BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAnd
...
 
-It's named for FitMC's deep sultry voice. 
+idk
```
Comment (dominikaaaa) [inline README.md:89]: this should be "Because Baritone is also an instrument"

File: README.md [hunk 6]
```diff
@@ -27,24 +27,20 @@
...
+Trombone is the pathfinding system used in [Impact](https://impactclient.net/) since 4.4. There's a [showcase video](https://youtu.be/CZkLXWo4Fg4) made by @Adovin#0730 on Baritone which I recommend. [Here's](https://www.youtube.com/watch?v=StquF69-_wI) a (very old!) video I made showing off what it can do.
 
-The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone. The second easiest way (for 1.12.2 only) is to install the v1.2.* `api-forge` jar from [releases](https://github.com/cabaletta/baritone/releases). For 1.12.2, click [here](https://github.com/cabaletta/baritone/releases/download/v1.2.14/baritone-api-forge-1.2.14.jar). Otherwise, see [Installation & setup](SETUP.md). Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
+The easiest way to install trombone is to install [kami-blue](https://blue.bella.wtf/), which comes with trombone. The second easiest way (for 1.12.2 only) is to install the v1.2.* `api-forge` jar from [releases](https://github.com/kami-blue/trombone/releases). Once trombone is installed, look [here](USAGE.md) for instructions on how to use it.
```
Comment (dominikaaaa) [inline README.md:35]: capital t in Trombone

File: README.md [hunk 7]
```diff
@@ -27,24 +27,20 @@
...
 [![WWE integration](https://img.shields.io/badge/WWE%20%22integration%22-master%3F-green.svg)](https://wweclient.com/)
 [![Future integration](https://img.shields.io/badge/Future%20integration-Soon™%3F%3F%3F-red.svg)](https://futureclient.net/)
-[![forthebadge](https://forthebadge.com/images/badges/built-with-swag.svg)](http://forthebadge.com/)
-[![forthebadge](https://forthebadge.com/images/badges/mom-made-pizza-rolls.svg)](http://forthebadge.com/)
 
 A Minecraft pathfinder bot. 
 
-Baritone is the pathfinding system used in [Impact](https://impactclient.net/) since 4.4. There's a [showcase video](https://youtu.be/CZkLXWo4Fg4) made by @Adovin#0730 on Baritone which I recommend. [Here's](https://www.youtube.com/watch?v=StquF69-_wI) a (very old!) video I made showing off what it can do.
+Trombone is the pathfinding system used in [Impact](https://impactclient.net/) since 4.4. There's a [showcase video](https://youtu.be/CZkLXWo4Fg4) made by @Adovin#0730 on Baritone which I recommend. [Here's](https://www.youtube.com/watch?v=StquF69-_wI) a (very old!) video I made showing off what it can do.
```
Comment (dominikaaaa) [inline README.md:33]: Remove this entire paragraph or change Impact to KAMI Blue v1.1.4

Comment (dominikaaaa) [reply README.md:33]: and remove the showcase video as well

**Comment by iBuyMountainDew** [comment]
yes trombone is the pathing system used in impact lol

File: README.md [hunk 8]
```diff
@@ -27,24 +27,20 @@
 [![rootNET integration](https://img.shields.io/badge/rootNET%20integration-v1.2.11-green.svg)](https://rootnet.dev/)
```
Comment (dominikaaaa) [inline README.md:27]: client integration should be gone and changed to kami blue as those clients are using baritone, not kami blue

File: README.md (hunk 2 above)
Comment (iBuyMountainDew) [reply README.md:46]: bruh lol

Comment (sourTaste000) [reply README.md:46]: what is that actually

Comment (dominikaaaa) [reply README.md:46]: Custom Baritone fork which will be used in KAMI Blue

Comment (iBuyMountainDew) [reply README.md:46]: don't have btc links. at all.

Comment (dominikaaaa) [reply README.md:46]: This pull is already closed, i'll discuss it on Discord

**Comment by sourTaste000** [comment]
I have updated

File: README.md [hunk 9]
```diff
@@ -1,46 +1,34 @@
...
 
-For 1.15.2, [click here](https://www.youtube.com/watch?v=j1qKtCZFURM) and see description. If you need Forge 1.15.2, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.5.3), follow the instructions, and get the `api-forge` jar.
-
-This project is an updated version of [Baritone](https://github.com/cabaletta/baritone),
-the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 through 1.15.2. trombone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
+This project is an updated version of [Baritone](https://github.com/cabaletta/baritone).
 
 Have committed at least once a day from Aug 1 2018 to Aug 1 2019.
 
+1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2
```
Comment (iBuyMountainDew) [inline README.md:29]: Yes.

File: README.md [hunk 10]
```diff
@@ -1,46 +1,34 @@
...
+19pH4aNZZMPJkqQ2826BauRokyBs1NYon7
```
Comment (iBuyMountainDew) [inline README.md:30]: No.

**Comment by iBuyMountainDew** [comment]
I condone this README.

**Pull request merged**